tests/components/ozw/test_websocket_api.py

```diff
@@ -119,6 +133,149 @@ async def test_websocket_api(hass, generic_data, hass_ws_client):
     assert result[2][ATTR_IS_AWAKE]
     assert not result[1][ATTR_IS_FAILED]
 
+    # Test get config parameters
+    await client.send_json({ID: 13, TYPE: "ozw/get_config_parameters", NODE_ID: 39})
+    msg = await client.receive_json()
+    result = msg["result"]
+    assert len(result) == 8
+    for config_param in result:
+        assert config_param["type"] in (
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6285789)